include/bluetooth/mesh/sensor.h

```diff
@@ -332,9 +336,9 @@ struct bt_mesh_sensor_format_cb {
 	 *  If @c val has a value that cannot be represented by the format,
 	 *  @c sensor_val will be set to the value clamped to the range
 	 *  supported by the format, and this function will return -ERANGE. This
-	 *  will clamp to "Greater than or equal to the maximum value" and "Less
-	 *  than or equal to the minimum value" if these are supported by the
-	 *  format.
+	 *  will clamp to "Greater than or equal to the maximum value" and
+	 *  "Less than or equal to the minimum value" if these are supported by
```

Comment:
If this is confusing anyone, apparently line-breaks inside quotes caused rendering issues (see e.g. how this paragraph is rendered in the [current latest documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/bluetooth_services/mesh/sensor.html#structbt__mesh__sensor__format__cb_1a4200878b7e4754a17a5282d23031773c))

Reply:
It looks a bit choppy... Haven't seen this before.
You tested that it looks normal when line breaks within quotes are removed?

Reply:
Yes, see doc preview: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-14234/nrf/libraries/bluetooth_services/mesh/sensor.html#c.bt_mesh_sensor_value_from_micro